app/destination/page.js

```diff
@@ -1,54 +1,73 @@
 "use client";
 
-import { useState } from 'react';
-
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
-
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
-  name,
-  onRemove,
-  thumbnail,
-}) => {
-  return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
-    </div>
-  );
-}
+import { useState } from "react";
 
+import styles from "@/components/destination/destination.module.css";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
+import PlanetWishlistItem from "@/components/destination/PlanetWishlistItem";
+import PlanetCard from "@/components/destination/PlanetCard";
 
 export const Destinations = () => {
+  const planets = [
+    {
+      name: "Europa",
+      description:
+        "Europa, one of Jupiter’s moons, is an icy world with a hidden ocean beneath its surface. This mysterious moon is a prime candidate for the search for extraterrestrial life, making it a thrilling destination for space explorers.",
+      thumbnail: "/destination/image-europa.png",
+    },
+    {
+      name: "Moon",
+      description:
+        "Our closest celestial neighbor, the Moon, is a silent witness to Earth's history. With its stunning craters and desolate landscapes, the Moon offers a unique glimpse into space exploration's past and future, making it a perfect destination for lunar adventurers.",
+      thumbnail: "/destination/image-moon.png",
+    },
+    {
+      name: "Mars",
+      description:
+        "Mars, the Red Planet, is a barren yet fascinating world with vast deserts, towering volcanoes, and the deepest canyon in the solar system. As humanity’s next frontier, Mars invites us to dream of colonization and the possibilities of life beyond Earth.",
+      thumbnail: "/destination/image-mars.png",
+    },
+    {
+      name: "Titan",
+      description:
+        "Titan, Saturn's largest moon, is a world of dense atmosphere and liquid methane lakes. This enigmatic moon is shrouded in a thick orange haze, concealing a landscape that is both alien and strangely familiar, beckoning explorers to uncover its secrets.",
+      thumbnail: "/destination/image-titan.png",
+    },
+  ];
   const [selectedPlanets, onAddPlanet] = useState([]);
```

Comment:
I know this was my mistake, but this could be called `[selectedPlanets, setSelectedPlanets]`

Reply:
Done